inlong-sort/sort-connectors/jdbc/src/main/java/org/apache/inlong/sort/jdbc/table/JdbcDialects.java

```diff
@@ -54,13 +58,22 @@ public static Optional<JdbcDialect> get(String url) {
         return Optional.empty();
     }
 
+    public static Optional<JdbcDialect> getCustomDialect(String dialectImpl) {
```

Comment:
is it better to put `getCustomDialect` wrapped in `get` func . Otherwise user sould know customDialect and normal dialect?

Reply:
The user does not need to perceive the difference between the two, but if the user specifies a certain dialect, we must use this dialect, and this effect cannot be achieved only by get, because the url may be the same.